multinet/swagger/template.yaml

```diff
@@ -14,35 +14,46 @@ definitions:
     required:
       - name
       - owner
-      - readers
+      - maintainers
       - writers
+      - readers
+      - public
     properties:
       name:
         description: The name of the workspace
         type: string
       owner:
-        description: The owner of the workspace, who has read and write access, and can also delete it
+        description: The owner of the workspace
         type: string
-      readers:
-        description: A list of users with read access to the workspace
+      maintainers:
+        description: The list of maintainers of the workspace
         type: array
         items:
           type: string
       writers:
-        description: A list of users with write access to the workspace
+        description: The list of writers of the workspace
+        type: array
+        items:
+          type: string
+      readers:
+        description: The list of readers of the workspace
         type: array
         items:
           type: string
+      public:
+        description: Whether this workspace is public or not
+        type: boolean
     example:
       name: engineering
       owner: laforge
       readers:
         - worf
         - data
         - troi
-      writers:
+      maintainers:
         - picard
         - riker
+      writers: []
```

Comment:
This example is missing the `public` flag

Reply:
Good catch. See d08ef44d9251f21c6e9e865a43c0323415d7fda2.

Of course, the example is already really wrong, because those entries should be `sub` values, not usernames.